app/src/api/types.ts

```diff
@@ -109,6 +109,21 @@ export namespace ApiData {
     validatorsCount: number;
     bidsCount: number;
   }
+
+  export interface ProcessedDeploy {
+    timestamp: number;
+    dateTime: Date;
+    deployHash: string;
+    blockHash: string;
+    publicKey: string;
+    action: string;
+    deployType: string | undefined;
```

Comment:
Is it required to be explicit with the undefined here?

Reply:
I'm not sure, this type was just copied over so I figured I didn't want to mess with it since I didn't write the initial code.